### DISCUSSION TITLE
How can I playback the HLS files of my live stream once they are uploaded to the AWS S3 bucket?

### DISCUSSION THREAD

**Discussion by tremzo** [ant-media/Ant-Media-Server#5620] [FAQs]
I've setup my Ant Media Server to upload all the HLS files of my live streams to my S3 bucket, but how can I use these HLS files to playback the live stream at a later time?

**Comment by Mohit-3196** [comment]
Hi @timantmedia 
Thank you for your question.

Once the HLS files are uploaded to the AWS S3 bucket, you can use the AWS S3 bucket object URL to access and play the HLS files.
The sample object URL is of the format `https://{bucket-name}.s3.{region-name}.amazonaws.com/streams/streamId.m3u8`
